data/input_files/default.lua

```diff
@@ -12,7 +12,7 @@ main_mesh = {
 }
 
 -- Solver parameters
-nonlinear_solid = {
+solid = {
     stiffness_solver = {
```

Comment:
So, when one is doing dynamics, isn't the "stiffness solver" really solving for a combination of mass+stiffness?

Maybe.  "solid mechanics solver" or simply "linear algebra"

Reply:
Yes. I remember thinking about this when it initially went in, and we couldn't think of a good name. Maybe `equatoin_solver` to match the internal class?

Reply:
Yeah equation_solver is a better name, because it is exactly what it is

Reply:
Ahhh I remember the problem. We don't use a lumped mass matrix, so we can also specify a mass matrix solver for explicit problems.

Reply:
However, they're not separate options after @samuelpmishLLNL 's ODE work, so I'll change it to `equation_solver`.